assets/css/style.css

```diff
@@ -1,3 +1,276 @@
-.makeHeaderRed {
-  color: red;
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: calc(20px + 0.390625vw);
+}
+
+body {
+  background-color: black;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 18px;
+  background-color: #3e3c3c;
+  box-shadow: 1px 2px 10px black;
+}
+
+.logo {
+  margin-left: 3%;
+}
+
+.hamburgerMenu {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  justify-content: center;
+}
+
+.nav-items {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 0;
+  padding-right: 30%;
+}
+
+.list-item {
+  list-style: none;
+  margin: 0;
+  padding: 0 25px;
+}
+
+.list-item a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.list-item a:hover,
+button:hover {
+  transform: translate(4px, 2px);
+  box-shadow: #ff6b00;
+}
+
+.hamburger-icon {
+  display: none;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  margin-right: 20px;
+}
+
+.hamburger-icon .bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: #e5e5e5;
+  border-radius: 2px;
+}
+
+.main-area {
+  background-image: url("../images/Illustration-header.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100vh;
+}
+
+.main-section {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  height: 100%;
+  font-family: 'Crete Round', serif;
+  margin-top: 4rem;
+}
+
+.main-header {
+  font-family: 'Crete Round', serif;
+  color: #ff6b00;
+  font-size: 33px;
+  margin-bottom: 1rem;
+}
+
+.second-header {
+  font-family: 'Roboto', serif;
+  color: #fff5e1;
+  margin-bottom: 1rem;
+  font-size: 25px;
+}
+
+.subtitle {
+  display: block;
+  margin-bottom: 0%;
+}
+
+.main-description {
+  font-family: 'Roboto', serif;
+  color: #fff5e1;
+  font-size: 20px;
+  margin-top: 0.6rem;
+}
```

Comment:
- [ ] For the description, it would be nice to use the CSS declarations from the Figma template that you chose. Using the same declarations would make your design look identical to that of the template. 💻 😎 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/219817645-b3799596-4932-4486-9a4c-4043665f2935.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/219817674-77d1ce01-2904-4ff5-b6c1-2f08a02d7045.png)

### CSS declarations

![image](https://user-images.githubusercontent.com/77106962/219817724-83d80352-06fd-4c20-8bdf-1b1fdd4ac57d.png)